content/docs/howto/configuration.md

```diff
@@ -252,6 +252,73 @@ For example, to make the BellSoft Liberica JRE dependency accessible available t
 
+### Dependency Mirrors
+Larger (corporate) networks might have a mirror server available to cache dependencies for access from within the local network. Dependency mirrors can be used to download all (or some) buildpack dependencies from such alternative locations regardless of their versions.
+
+If dependency mirrors and dependency mappings are defined at the same time, those artifacts specifically mapped as described in [Dependency Mappings]({{< relref "#dependency-mappings" >}}) are loaded accordingly. All other dependencies are downloaded from the mirror, should one apply.
+
+Mirrors can be defined in two ways.
+1. Setting the `BP_DEPENDENCY_MIRROR` environment variable(s).
+2. Including a binding with a type of `dependency-mirror`.
+
+#### Setting Default Mirror
+In most cases, it is sufficient to set just one mirror from which all dependencies should be downloaded.  
+
+**Example**: Using Environment Variable
+
+Using the environment variable `BP_DEPENDENCY_MIRROR=https://mirror.example.org` would override the original URIs and download all dependencies from this host, whilst preserving the original paths.
+
+Let's assume, we have a dependency with the original URI of `https://github.com/bell-sw/Liberica/releases/download/11.0.8+10/bellsoft-jre11.0.8+10-linux-amd64.tar.gz`.  
+This setting would download the dependency from `https://mirror.example.org/bell-sw/Liberica/releases/download/11.0.8+10/bellsoft-jre11.0.8+10-linux-amd64.tar.gz`.
+
+**Example**: Using a Binding
+
+Instead of using the environment variable, we could achieve the same by setting a binding of type `dependency-mirror` using the `default` key and the mirror URI as the file content.
+```
+/platform
+    └── bindings
+        └── dependency-mirror
+            ├── default                https://mirror.example.org
+            └── type                   dependency-mirror
+```
+
+#### Path Prefix and Hostname Placeholder
+The mirror URI may also include a prefix. Using the above example, a value of `https://mirror.example.org/buildpack-dependencies` would lead to downloads from `https://mirror.example.org/buildpack-dependencies/bell-sw/Liberica/releases/download/11.0.8+10/bellsoft-jre11.0.8+10-linux-amd64.tar.gz`.  
+
+Similarly, including the placeholder `{originalHost}` as in `https://mirror.example.org/{originalHost}` would preserve the original URI's hostname and download from `https://mirror.example.org/github.com/bell-sw/Liberica/releases/download/11.0.8+10/bellsoft-jre11.0.8+10-linux-amd64.tar.gz`.  
+
+This placeholder can also be used together with a path prefix. E.g.: `https://mirror.example.org/buildpack-dependencies/{originalHost}`
+
+#### Schemes and Authentication
+Dependency mirror URIs can either use the `https` or `file` schemes and include basic authentication credentials, should the mirror require them.  
+The credentials can be passed to the server using the format `https://[username]:[password]@mirror.example.org`.
+
+#### Setting Hostname Mirrors
```

Comment:
Could you add an example of how a URL would be transformed by each of the settings for clarity?

Reply:
I've tried to add a little more detail on how the URL transformation takes place when hostname mirrors are used.
Please have a look and let me know if this brings more clarity.